new/LIT7017Crucifixion.xml

```diff
@@ -0,0 +1,80 @@
+                <title xml:lang="gez" xml:id="t3">Salām la-śaqāyka la-ʾaḥadu-ʾaḥadu</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient8819"/>
+                    <witness corresp="DimmaUnesco1015"/>
+                    <witness corresp="ESdd017"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Chants"/>
+                    <term key="ChristianContent"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-06-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following incipit is based on <ref type="mss" corresp="BLorient8819"/>.</note>
+                <div type="textpart" corresp="incipit" xml:lang="gez">
+                <ab>
+                    ሰላም፡ ለሥቃይከ፡ ለለ፩፩፤
+                    <lg n="1" type="stanza" xml:id="s1">
+                    <l n="1">እምሴተ፡ ሐሙስ፡ ዘኮነ፡ እስከ፡ ምሴተ፡ ዓርብ፡ የዓዱ።</l>
+                    <l n="2">ኢየሱስ፡ ክርስቶስ፡ ተስፋ፡ ኤዎስጣቴዎስ፡ ወውሉዱ።</l>
+                    <l n="3">ዘበእንቲአሁ፡ ሶበ፡ በውስቴቱ፡ ወረዱ፤</l>
+                    <l n="4">ለእሳተ፡ ጤገን፡ ዘብርት፡ አብሰሎሙ፡ ነዱ።</l>
+                    </lg>
+                </ab>
+                </div>
+                </div><!---->
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT7017Crucifixion" passive="LIT4035SenkessarDL#Mask27SalEwostatewos"/>
+                </listRelation>
```

Comment:
Is it a part of the salam or the entire salam `LIT4035SenkessarDL#Mask27SalEwostatewos`? 
If the entire then it should rather be, in `LIT4035SenkessarDL`
```
<div type="textpart" subtype="commemoration" xml:id="Mask27SalEwostatewos" corresp="LIT7017Crucifixion">
                        <label>Salām-hymn to ʾEwosṭātewos</label>
                     </div>
```